features/staging_commands.feature

```diff
@@ -264,74 +266,6 @@ Feature: Enable command behaviour when attached to an UA staging subscription
         """
 
         Examples: ubuntu release
-        | release | next_release | fips-service  | fips-name    |
-        | xenial  | bionic       | fips          | FIPS         |
-
-   @series.xenial
-   @uses.config.machine_type.lxd.vm
-   Scenario Outline: Attached FIPS upgrade across LTS releases
```

Comment:
Was this a complete duplicate test? if so let's mention the reason for dropping it in your PR commit message

Reply:
It was not exactly a duplicate. This is the test that was asserting that `fips-updates` ended up with `ubuntu-fips` uninstalled after a upgrade from xenial to bionic. Since this behavior is not happening, I have integrated the test with the fips one